content/docs/command-reference/get.md

```diff
@@ -36,14 +36,13 @@ the data source. Both HTTP and SSH protocols are supported for online repos
 to an "offline" repo (if it's a DVC repo without a default remote, instead of
 downloading, DVC will try to copy the target data from its <abbr>cache</abbr>).
 
-The `path` argument of this command is used to specify the location of the
-target to be downloaded within the source repository at `url`. `path` can
-specify any file or directory in the source repo, including <abbr>outputs</abbr>
-tracked by DVC, as well as files tracked by Git. Note that for DVC repos, the
-target should be found in one of the
-[DVC-files](/doc/user-guide/dvc-file-format) of the project. The project should
-also have a default [DVC remote](/doc/command-reference/remote), containing the
-actual data.
+The `path` argument is used to specify the location of the target to be
+downloaded within the source repository at `url`. `path` can specify any file or
+directory in the source repo, including <abbr>outputs</abbr> tracked by DVC, as
```

Comment:
can we avoid the outputs term here? I would say in general, we should be using outputs only when we deal with pipelines. I would avoid using this term when we apply it to a `dvc added` file

Reply:
OK. Yes, it makes sense. I need to review all the instances of this glossary term then ⏳

Reply:
Is "data artifact" a better term here? Or should that also be avoided? Because in the glossary it's basically a synonym of "outputs" currently and I've been thinking about merging them or getting rid of "data artifact"... Maybe this is my chance!

Reply:
(Maybe this is ~~my~~ the chance) to redefine it a little so it's useful for non stage/pipeline files/dirs traced by DVC (found in `outs` fields of DVC-files)

Reply:
data files, directories, models 

`outs` field is a technical impl detail

Reply:
Hmmm but we are often saying "found insid a DVC-file" so specifying "in the `outs` field" isn't a bad idea, plus the command reference is supposed to have technical details, no?

Reply:
It's fine when we specifically describe _how_ we do something. I would avoid mentioning this when we describe _what_ we do, the essence of something from the user perspective.

Reply:
OK well I removed it form here and other related files in 90befbd1 but I still need to review the rest of the references. Resolving here. I'll request your review ASAP.